File: web/src/api/memUserSafe.ts
import { http,resData } from "@/utils/http";
 
// @Tags MemUserSafe
// @Summary 创建MemUserSafe
export const createMemUserSafe = (data?: object) => {
   return http.request<resData>("post", "/memUserSafe/createMemUserSafe", { data });  
}

// @Tags MemUserSafe
// @Summary 删除MemUserSafe
export const deleteMemUserSafe = (data?: object) => {
   return http.request<resData>("post", "/memUserSafe/deleteMemUserSafe", { data });  
}
 

// @Tags MemUserSafe
// @Summary 删除ByIdsMemUserSafe
export const deleteMemUserSafeByIds = (data?: object) => {
   return http.request<resData>("post", "/memUserSafe/deleteMemUserSafeByIds", { data });  
}
 

// @Tags MemUserSafe
// @Summary 更新MemUserSafe
export const updateMemUserSafe = (data?: object) => {
   return http.request<resData>("post", "/memUserSafe/updateMemUserSafe", { data });  
}

// @Tags MemUserSafe
// @Summary 快速编辑
export const quickEdit = (data?: object) => {
   return http.request<resData>("post", "/memUserSafe/quickEdit", { data });  
}
 

// @Tags MemUserSafe
// @Summary 用id查询MemUserSafe
export const findMemUserSafe = (params?: object) => {
   return http.request<resData>("get", "/memUserSafe/findMemUserSafe", { params });  
}
 

// @Tags MemUserSafe
// @Summary 分页获取MemUserSafe列表
export const getMemUserSafeList = (params?: object) => {
   return http.request<resData>("get", "/memUserSafe/getMemUserSafeList", { params });  
}
 

// @Tags MemUserSafe
// @Summary 导出excelMemUserSafe列表
export const excelList = (params?: object) => {
   return http.request<resData>("get", "/memUserSafe/excelList", { params });  
}

File: server/router/business/memUserSafe.go
package business

import (
	"go-cms/api/v1"
	"go-cms/middleware"
	"github.com/gin-gonic/gin"
)

type MemUserSafeRouter struct {
}

// InitMemUserSafeRouter 初始化 MemUserSafe 路由信息
func (s *MemUserSafeRouter) InitMemUserSafeRouter(Router *gin.RouterGroup) {
	router := Router.Group("memUserSafe").Use(middleware.OperationRecord())
	routerNoRecord := Router.Group("memUserSafe")
	var apiV1 = v1.ApiGroupApp.BusinessApiGroup.MemUserSafeApi
	{
		router.POST("createMemUserSafe", apiV1.CreateMemUserSafe)   // 新建MemUserSafe
		router.POST("deleteMemUserSafe", apiV1.DeleteMemUserSafe) // 删除MemUserSafe
		router.POST("deleteMemUserSafeByIds", apiV1.DeleteMemUserSafeByIds) // 批量删除MemUserSafe
		router.POST("updateMemUserSafe", apiV1.UpdateMemUserSafe)    // 更新MemUserSafe
	    router.POST("quickEdit", apiV1.QuickEdit)  // 快速编辑	
	}
	{
		routerNoRecord.GET("findMemUserSafe", apiV1.FindMemUserSafe)        // 根据ID获取MemUserSafe
		routerNoRecord.GET("getMemUserSafeList", apiV1.GetMemUserSafeList)  // 获取MemUserSafe列表
        routerNoRecord.GET("excelList", apiV1.ExcelList)  // 分页导出excel MemUserSafe列表
	}
}
